server/account/middleware.ts

```diff
@@ -38,9 +91,14 @@ const isValidUsername = (req: Request, res: Response, next: NextFunction) => {
 /**
  * Checks if a password in req.body is valid, that is, at 6-50 characters long without any spaces
  */
-const isValidPassword = (req: Request, res: Response, next: NextFunction) => {
+const isValidPassword = (required: boolean) => (req: Request, res: Response, next: NextFunction) => {
+  if (!required && req.body.password === undefined) {
+    next();
+    return;
+  }
+  
   const passwordRegex = /^\S+$/;
```

Comment:
I believe the current behavior of the password checking does not reflect the character length restrictions

Reply:
very true. This is just copied from the Fritter starter code haha. I'll just get rid of the restriction for now. If we want password requirements we can add later.